src/ts/order.ts

```diff
@@ -297,11 +299,25 @@ export function hashOrder(domain: TypedDataDomain, order: Order): string {
 export function hashOrderCancellation(
   domain: TypedDataDomain,
   orderUid: BytesLike,
+): string {
```

Comment:
Should we annotate with:

```ts
/**
 * @deprecated The method should not be used
 */
 ```

...since this will become deprecated soon?

Reply:
I didn't as I re-implemented the signatures here to work with the new API. (i.e. `hashOrderCancelation(uid) == hashOrderCancellations([uid])`.

Reply:
Makes sense!